components/usage/pkg/controller/reconciler.go

```diff
@@ -131,16 +131,16 @@ func (u *UsageReconciler) ReconcileTimeRange(ctx context.Context, from, to time.
 func generateUsageReport(teams []teamWithWorkspaces, maxStopTime time.Time) ([]TeamUsage, error) {
 	var report []TeamUsage
 	for _, team := range teams {
-		var teamTotalRuntime time.Duration
+		var teamTotalRuntime float64
```

Comment:
Why the overkill floating-point precision on seconds? 🤔 Rounded-up seconds would probably be precise enough, right?

Reply:
Credits are likely to map to hours anyway. I think rounding to the next second is extremely fine. 👍

Reply:
Let's do that in a follow-up PR.